skietbaan_fe/src/components/RegisterStyles.css

```diff
@@ -177,20 +177,39 @@
 
 .welcome-header{
   width: 100%;
-  height: 80px;
-  background-color: lightgray;
+  height: 79px;
+  background-color: #EA241A;
   text-align: center;
 }
 
 .welcome-label{
+  text-align: center;
+  padding-top: 9px;
+  font-size: 21px;
+  color: white;
+  font-weight: 100;
+  float: right;
+  padding-right: 24px;
+}
+
+.skietbaan-label{
   text-align: center;
   padding-top: 23px;
-  font-size: 22px;
-  color: #e11e2b;
+  font-size: 19px;
+  color: #EA241A;
 }
 
 .bullet-button{
   height: 14px;
   margin-left: 10px;
   margin-top: -4px;
 }
+
+.hidden{
+  display: none;
+}
+
+.header-image{
+  height: 94px;
+  width: 100%;
```

Comment:
Read this:
https://stackoverflow.com/questions/17468733/difference-between-width-auto-and-width-100-percent

Reply:
Thanks read it but auto is not what I need to use in this case

Reply:
nope, was just a note